src/registrar/models/domain.py

```diff
@@ -1100,14 +1102,28 @@ def is_expired(self):
         now = timezone.now().date()
         return self.expiration_date < now
 
-    def state_display(self):
+    def is_expiring(self):
```

Comment:
Is the 60 days window  a precedent set by Cloudflare or outside sources -- ie is there a reason our expiration window is 60 days? Curious if we should set a param for an expiration window that we default set to 60

Reply:
@erinysong It isn't set by Cloudflare, its set by us(as part of our ac). Do you see an instance where that the expiration window would change?